modules/installation-configuration-parameters.adoc

```diff
@@ -577,17 +577,17 @@ accounts for the dramatically decreased machine performance.
 |Array of `MachinePool` objects.
 
 ifndef::openshift-origin[]
-ifndef::aws,bare,ibm-z,ibm-power,azure,ibm-power-vs[]
+ifndef::aws,bare,gcp,ibm-z,ibm-power,azure,ibm-power-vs[]
 |`controlPlane.architecture`
 |Determines the instruction set architecture of the machines in the pool. Currently, clusters with varied architectures are not supported. All pools must specify the same architecture. Valid values are `amd64` (the default).
 |String
-endif::aws,bare,ibm-z,ibm-power,azure,ibm-power-vs[]
+endif::aws,bare,gcp,ibm-z,ibm-power,azure,ibm-power-vs[]
 
-ifdef::aws,bare,azure[]
+ifdef::aws,bare,azure,gcp[]
 |`controlPlane.architecture`
 |Determines the instruction set architecture of the machines in the pool. Currently, clusters with varied architectures are not supported. All pools must specify the same architecture. Valid values are `amd64` and `arm64`. See _Supported installation methods for different platforms_ in _Installing_ documentation for information about instance availability.
```

Comment:
Out of scope, really, but maybe you could replace the sentence with an xref to https://docs.openshift.com/container-platform/4.13/installing/installing-preparing.html#supported-installation-methods-for-different-platforms?

Reply:
I would want to but Ik we can't pus xrefs in modules :'(